chain/postgresdb/database/database.go

```diff
@@ -12,25 +12,11 @@ import (
 	"time"
 
 	"github.com/jmoiron/sqlx"
-	"github.com/lib/pq" // Calls init function.
+	_ "github.com/lib/pq" // Calls init function.
```

Comment:
This doesn't seem great: https://github.com/lib/pq#status
Why not use a maintained package?

Reply:
This module is done and it works. I don't need to worry about changes or bugs. The pgx package doesn't have a stable drive IMHO, and I like using sqlx for this. I am using this in other production systems without issue.

In the end, I like the idea of a module being done and not needing more.

Reply:
I guess this one is still maintained so it's fine for now.